Cargo.toml

```diff
@@ -86,6 +86,8 @@ tracing = "0.1"
 tracing-subscriber = "0.2"
 url = "2.1"
 
+dashmap = { version = "4.0.2", features = ["raw-api"] }
```

Comment:
can you add it to the alphabetically ordered list above? 😉

Reply:
Done!